### PR TITLE
Feature 126 sosmeta converter

### PR DESCRIPTION
**Pull request by villejuutila**
Contains: 

- Resolve of #126 
- Minor changes to draft-editor.screen to better support the functionality of this feature

### CODE REVIEW

File: src/components/editor/editor-screen-drawer.tsx [hunk 1]
```diff
@@ -52,14 +56,36 @@ const EditorScreenDrawer: FC<Props> = ({
...
+    try {
+      const convertedForm = await SosmetaUtils.convertSosmetaToMetaform(formSettings.formSchema);
+
+      setFormSettings({
+        ...formSettings,
+        formName: convertedForm!.title!,
+        formSections: convertedForm!.sections!
+      });
+    } catch (e) {
+      console.log(e);
```
Comment (EVP1989) [inline src/components/editor/editor-screen-drawer.tsx:74]: Do proper error handling

File: src/utils/sosmeta-utils.ts [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+ */
+enum SosmetaType {
+  ARRAY = "array",
+  BOOLEAN = "boolean",
+  INTEGER = "integer",
+  NULL = "null",
+  NUMBER = "number",
+  OBJECT = "object",
+  STRING = "string"
+}
```
Comment (EVP1989) [inline src/utils/sosmeta-utils.ts:17]: Move to types

**Review by EVP1989 — COMMENTED** [review]
Added some comments

File: src/components/editor/editor-screen-drawer.tsx (hunk 1 above)
Comment (villejuutila) [reply src/components/editor/editor-screen-drawer.tsx:74]: Done

**Review by EVP1989 — APPROVED** [review]
Approved

**Pull request merged**